canonical-web-and-design/maas-ui
# feat(ui): add action to navigate to edit aliases and vlans

## Done

- Update the network table action to navigate to the form for editing aliases and vlans.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the network tab for a machine.
- On different interface types open the action menu and click edit.
- The form title should match the interface type and you should see the interface form for physical interfaces and just the table and a close button for all others.

## Fixes

Fixes: #2223.
